packages/db-react/src/options-network.tsx

```diff
@@ -30,7 +30,7 @@ export const optionsNetwork = {
       onSuccess: () => toastSuccess('Network deleted'),
       ...props,
     }),
-  findMany: (input: NetworkFindManyInput = {}) =>
+  findMany: (input: NetworkFindManyInput) =>
```

Comment:
Updated these inputs based on [this comment](https://github.com/samui-build/samui-wallet/pull/689#discussion_r2588668322)